### DISCUSSION TITLE
Amazing Compression!!!!

### DISCUSSION THREAD

**Discussion by nnw522** [FormerLurker/ArcWelderPlugin#212] [Show and tell]
![Screenshot (1)](https://user-images.githubusercontent.com/84998259/130003331-31990401-0573-4d71-966f-9bdcd03095cc.png)

Do I need to say anymore? Simply Amazing piece of software!!!!

**Comment by FormerLurker** [comment]
Thank you so much for letting me know!  You made my day 🙂.

If you want, maybe try the latest development version?  I am going to pish a few a few more changes this weekend with the goal of pushing out a release before the end of the month.  Could use some testers if you are interested.

**Comment by nnw522** [comment]
I would love to participate.

**Comment by FormerLurker** [comment]
Excellent! I'll reply here after the updates are completed.  Should be tomorrow or sat.

**Comment by nnw522** [comment]
Awesome, thanks!

**Comment by FormerLurker** [comment]
So, life intervened, and I wasn't able to finish these changes.  Gonna be putting in time in the evenings until things are ready.  Sorry for the delays!

**Comment by nnw522** [comment]
No worries, thanks for the heads up!
